sdk/cognitiveservices/cognitiveservices-luis-runtime/README.md

```diff
@@ -17,140 +17,127 @@ npm install @azure/cognitiveservices-luis-runtime
 
 #### nodejs - Authentication, client creation and getVersionPrediction prediction as an example written in TypeScript.
 
-##### Install @azure/ms-rest-azure-js
+##### Install @azure/ms-rest-nodeauth
 
+- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
 ```bash
-npm install @azure/ms-rest-azure-js
+npm install @azure/ms-rest-nodeauth@"^3.0.0"
 ```
 
 ##### Sample code
 
 ```typescript
-import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
-import { LUISRuntimeClient } from "@azure/cognitiveservices-luis-runtime";
-
-let authoringKey = process.env["luis-authoring-key"];
-const creds = new CognitiveServicesCredentials(authoringKey);
-
-// check the following link to find your region
-// https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-regions
-const region = "<your-region>";
-const client = new LUISRuntimeClient(creds, "https://" + region + ".api.cognitive.microsoft.com/");
-
-const appId = "<your-app-id>"; // replace this with your appId.
-const versionId = "0.1"; // replace with version of your luis application. Initial value will be 0.1
-
-const predictionRequest = {
-  query: "testquery",
-  options: {
-    datetimeReference: new Date(),
-    overridePredictions: true
-  },
-  externalEntities: [
-    {
+import * as msRest from "@azure/ms-rest-js";
```

Comment:
The auto-generated code sample in the read me file does not work well for cognitive services. We spent some time last month cleaning up the samples across all the cognitive services libraries.

Please revert the changes to this file, and go over the code snippet manually to update only what has changed in your api